var/spack/repos/builtin/packages/py-fparser/package.py

```diff
@@ -8,23 +8,27 @@
 
 class PyFparser(PythonPackage):
     """
-    This project is based upon the Fortran (77..2003) parser
+    This project is based upon the Fortran parser
     originally developed by Pearu Peterson for the F2PY project,
     www.f2py.com. It provides a parser for Fortran source code
-    implemented purely in Python with minimal dependencies.
+    (up to and including F2008) implemented purely in Python with
+    minimal dependencies.
 
     """
 
     # Links
     homepage = "https://github.com/stfc/fparser"
     git = "https://github.com/stfc/fparser.git"
-    pypi = "fparser/fparser-0.1.4.tar.gz"
+    pypi = "fparser/fparser-0.2.0.tar.gz"
+
+    maintainers = ["arporter"]
 
     # License
     license("BSD-3-Clause")
 
     # Releases
     version("develop", branch="master")
+    version("0.2.0", tag="v0.2.0")
```

Comment:
Avoid mixing version types. `git` version downloads are slower and can't be cached as easily. Missing commit on tag means there is no checksum to guard against changing code.
```suggestion
    version("0.2.0", sha256="3901d31c104062c4e532248286929e7405e43b79a6a85815146a176673e69c82")
```

Reply:
Oh yes, thanks.